app/src/main/java/ch/hikemate/app/ui/navigation/NavigationActions.kt

```diff
@@ -36,7 +36,7 @@ data class TopLevelDestination(val route: String, val icon: ImageVector, val tex
 
 /** Object containing top-level destinations. */
 object TopLevelDestinations {
-  val PLANNED_HIKES = TopLevelDestination(Route.PLANNED_HIKES, Icons.Outlined.Menu, "Planned Hikes")
+  val PLANNED_HIKES = TopLevelDestination(Route.PLANNED_HIKES, Icons.Outlined.Home, "Planned Hikes")
```

Comment:
Question: Is "home" a house icon? Does that fit?

Reply:
It fits for now. I had the `Menu` Icon for both the `SideBarNavigation` and for the `Planned_Hikes` screen so I changed it. In the future we might need to create one of our own.